src/app/static/src/app/components/adr/ADRIntegration.vue

```diff
@@ -5,7 +5,7 @@
             <select-dataset></select-dataset>
             <div class="pt-3" id="adr-capacity" v-if="selectedDataset">
                 <span class="font-weight-bold align-self-stretch" v-translate="'adrAccessLevel'"></span>
-                <span v-tooltip="handleUploadPermission(hasUploadPermission, true)">
+                <span v-tooltip="handleUploadPermission(hasUploadPermission, true)" style="margin-left: 4px;">
```

Comment:
can you use bootstrap spacing such as `ml-4` instead?

Reply:
done